README.md

```diff
@@ -2,6 +2,8 @@
 
 Raspberry Pi Imaging Utility
 
+Downloaded prebuilt versions from https://www.raspberrypi.org/downloads/
```

Comment:
- The page title (first line of the file) should start with `# `, which makes it a level 1 heading, for consistency with other repos. (Also, level 2,3 and 4 headings are used further down the page).

- Downloaded -> Download

- I can't find prebuilt or pre-built in any of the free online dictionaries, but I would imagine pre-built would nevertheless be correct.

- URL should be tagged in markdown to linkify it - that also lets you hide the http:// part.

- Sentence should finish with a full stop

Reply:
Can't believe I made such a silly "downloaded" typo - my fingers must have been tired! :laughing:

Reply:
> The page title (first line of the file) should start with #

It already does :stuck_out_tongue_winking_eye:

Reply:
> > The page title (first line of the file) should start with #
> 
> It already does 😜

My mistake 🙃